usage/jsgui/src/main/webapp/index.html

```diff
@@ -26,6 +26,10 @@
 
     <title>Brooklyn JS REST client</title>
 
+	<link rel="stylesheet" href="/assets/css/codemirror.css">
+    <link rel="stylesheet" href="/assets/css/addon/hint/show-hint.css">
+    <link rel="stylesheet" href="/assets/css/complete.css">
```

Comment:
+ these files should not be referenced here. They should be included in [styles.css](https://github.com/apache/incubator-brooklyn/blob/master/usage/jsgui/src/main/webapp/assets/css/styles.css) so that they can be concatenated into one file at build time.